closed/autoconf/custom-spec.gmk.in

```diff
@@ -56,7 +56,7 @@ WARNING_MODULES := \
 	#
 
 ifneq (,$(filter $(WARNING_MODULES),$(MODULE)))
-  JAVA_WARNINGS_ARE_ERRORS :=
+  JAVA_WARNINGS_AS_ERRORS := false
```

Comment:
Does this apply to OpenJ9 JCL code? Since `java-warnings-as-errors` is true by default, there are quite a few previous warning that are errors now.

Reply:
This applies to the same set of modules as before (including OpenJ9 JCL code).
The change in `DDR.gmk` is extra for compiling source for `openj9.dtfj`.